pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -292,6 +293,8 @@ func (r *ReconcileHyperConverged) Reconcile(request reconcile.Request) (reconcil
 	for _, f := range []func(*hcov1alpha1.HyperConverged, logr.Logger, reconcile.Request) error{
 		r.ensureKubeVirtConfig,
 		r.ensureKubeVirtStorageConfig,
+		r.ensureKubeVirtStorageRole,
+		r.ensureKubeVirtStorageRoleBinding,
```

Comment:
Why have we to do it at runtime via HCO?
Isn't OLM not enough for this?

Reply:
I assume it can. I wasn't sure about how I'm expected to do something like this.
HCO seems to work with kubernetes too, whereas OLM is apparently specific to OpenShift.
Are we expected to add to the OLM files and have the kubernetes bits be generated?

Reply:
Yes, something like that:
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/deploy/cluster_role_binding.yaml and https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/deploy/cluster_role.yaml are used only on plain k8s deployment.

https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/deploy/olm-catalog/kubevirt-hyperconverged/1.1.0/kubevirt-hyperconverged-operator.v1.1.0.clusterserviceversion.yaml is instead the CSV for OLM based deployment.

both the set of files are generated running `hack/build-manifests.sh` according to what's provided by the components operators.
In order to consume a different component you have to patch go.mod and run `go mod vendor` and `hack/config`

Reply:
In this case HCO is adding the ConfigMap, so it feels like the role should be something HCO adds. Is there somewhere that isn't auto-generated for appending to HCO-specific additions?